src/Nova/Booking.php

```diff
@@ -56,63 +52,75 @@ public function fieldsForIndex(NovaRequest $request)
     {
         return array_filter([
             ID::make()->sortable(),
-            Text::make('Room', 'room.id', function () {
-                return $this->room->name;
-            })->sortable(),
-            Text::make('Game Start', 'slot.start_at', function () {
-                return $this->slot->formatted_start;
-            })->sortable(),
-            Date::make('Game Date', 'slot.date')->sortable(),
-            Number::make('Participants'),
-            DateTime::make('Purchased', 'created_at')->sortable(),
+
+            Text::make('Slot Number')->sortable(),
+
+            Number::make('Total Participants')->sortable(),
+
+            MorphTo::make('Experience')->types([
+                nova('escaperoom_game')
```

Comment:
Do we have to list all of the morph types?

Reply:
Do you mean Escaperoom Market, Escaperoom Location, Escaperoom Rate, Escaperoom Theme?